Bottom: 4778a7cc361703125f3424a288425864a7f52e4d
Top:    d0ef1fc195dd74b5c969a834701bd09677452318
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-12 13:58:09 -0500

Add 'restore' tag to tasks

By tagging the tasks, restoration can be skipped using --skip-tags
option to ansible-playbook. This is useful for testing ansible plays
on systems without tapedrives or established backups.



---

diff --git a/roles/bacula-catalog/tasks/main.yml b/roles/bacula-catalog/tasks/main.yml
index 2d22328..3deb3fa 100644
--- a/roles/bacula-catalog/tasks/main.yml
+++ b/roles/bacula-catalog/tasks/main.yml
@@ -4,3 +4,4 @@
   register: filecount
 - include_tasks: restore-catalog-with-bsr.yml
   when: filecount.stdout_lines[1] == "0"
+  tags: restore
diff --git a/roles/docker-buildsystem/tasks/main.yml b/roles/docker-buildsystem/tasks/main.yml
index b62739e..382a979 100644
--- a/roles/docker-buildsystem/tasks/main.yml
+++ b/roles/docker-buildsystem/tasks/main.yml
@@ -133,3 +133,4 @@
 
 - import_tasks: restore.yml
   when: st_buildsystem_restore.stat.exists == False
+  tags: restore
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index 5240723..16da917 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -117,6 +117,7 @@
 
 - include_tasks: restore.yml
   when: st_exim4_restore.stat.exists == False
+  tags: restore
 
 # *****************************************************************************
 # Email accounts
diff --git a/roles/docker-gitlab/tasks/main.yml b/roles/docker-gitlab/tasks/main.yml
index 50f71c6..e75e328 100644
--- a/roles/docker-gitlab/tasks/main.yml
+++ b/roles/docker-gitlab/tasks/main.yml
@@ -181,3 +181,4 @@
 
 - include_tasks: restore.yml
   when: st_gitlab_restore.stat.exists == False
+  tags: restore
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index df44519..b77fcd4 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -186,3 +186,4 @@
 
 - include_tasks: restore.yml
   when: st_mantisbt_restore.stat.exists == False
+  tags: restore
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 251ba34..e1b0bfc 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -147,3 +147,4 @@
 
 - include_tasks: restore.yml
   when: st_mediawiki_restore.stat.exists == False
+  tags: restore
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index e771b77..20d8144 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -145,6 +145,7 @@
 
 - include_tasks: restore.yml
   when: st_openldap_restore.stat.exists == False
+  tags: restore
 
 # *****************************************************************************
 # Set the LDAP passwords
diff --git a/roles/docker-openssl/tasks/main.yml b/roles/docker-openssl/tasks/main.yml
index 7bb3bf8..a3828f6 100644
--- a/roles/docker-openssl/tasks/main.yml
+++ b/roles/docker-openssl/tasks/main.yml
@@ -97,3 +97,4 @@
 
 - include_tasks: restore.yml
   when: st_openssl_restore.stat.exists == False
+  tags: restore
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index f87afb7..5503d87 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -98,3 +98,4 @@
 
 - include_tasks: restore.yml
   when: st_svn_restore.stat.exists == False
+  tags: restore
